e2e-tests/gef/cypress/integration/exporters-incomplete-details.spec.js

```diff
@@ -0,0 +1,84 @@
+import applicationDetails from './pages/application-details';
+import dashboardPage from './pages/dashboard-page';
+import companiesHouse from './pages/companies-house';
+import automaticCover from './pages/exporters-address';
+
+import CREDENTIALS from '../fixtures/credentials.json';
+
+let url;
+let dealId;
+
+context('Incomplete exporter section - application details page', () => {
+  describe('Creating application and checking if incomplete exporter sections are showing', () => {
+    beforeEach(() => {
+      cy.login(CREDENTIALS.MAKER);
+    });
+
+    it('creates the application', () => {
+      dashboardPage.createNewSubmission().click();
+      dashboardPage.gefSubmission().click();
+      dashboardPage.continueButton().click();
+      dashboardPage.mandatoryCriteriaYes().click();
+      dashboardPage.continueButton().click();
+      dashboardPage.internalRefName().type('A');
+      dashboardPage.continueButton().click();
+      cy.url().then((thisUrl) => {
+        url = thisUrl;
+        // get dealId from last split
+        dealId = thisUrl.split('/').slice(-1);
+      });
+    });
+
+    it('add the exporter', () => {
+      cy.visit(url);
+      applicationDetails.exporterDetailsLink().click();
+      companiesHouse.regNumberField().type('8989898');
```

Comment:
is this a mock value?

Reply:
no, used the example number from description